### PR TITLE
Fix for issue: 106, relative paths for coded tests

### PR DESCRIPTION
**Pull request by johnpaliotta**
The only way to get relative paths is to edit the .tst so this is a low priority issue.

### CODE REVIEW

**Comment by johnpaliotta** [comment]
@Zbigor   please review

File: python/vTestInterface.py [hunk 1]
```diff
@@ -136,7 +136,7 @@ def getPassFailString(test):
     return XofYString(numerator, denominator)
 
 
-def generateTestInfo(test):
+def generateTestInfo(enviroPath, test):
```
Comment (aytey) [inline python/vTestInterface.py:139]: Need to update the docstring

File: python/vTestInterface.py [hunk 2]
```diff
@@ -155,8 +155,10 @@ def generateTestInfo(test):
     if vpythonHasCodedTestSupport and test.coded_tests_file:
         # guard against the case where the coded test file has been renamed or deleted
         # or dataAPI has a bad line nuumber for the test, and return None in this case.
-        if os.path.exists(test.coded_tests_file) and test.coded_tests_line > 0:
-            testInfo["codedTestFile"] = test.coded_tests_file
+        enclosingDirectory = os.path.dirname (enviroPath)
```
Comment (aytey) [inline python/vTestInterface.py:158]: Please run `black`

**Comment by aytey** [comment]
@johnpaliotta: do we need a test for this? How do we get a relative file path for a coded test?

File: python/vTestInterface.py (hunk 2 above)
Comment (johnpaliotta) [reply python/vTestInterface.py]: > Please run `black`

done

File: python/vTestInterface.py (hunk 1 above)
Comment (johnpaliotta) [reply python/vTestInterface.py:139]: > Need to update the docstring

Done

**Comment by johnpaliotta** [comment]
> @johnpaliotta: do we need a test for this? How do we get a relative file path for a coded test?

Probably should have a test.
As I said in the first comment above :)  ... you have to manually edit the .tst file.

**Pull request merged**